agent/config.go

```diff
@@ -2023,6 +2036,9 @@ func MergeConfig(a, b *Config) *Config {
 	if b.ACLDefaultPolicy != "" {
 		result.ACLDefaultPolicy = b.ACLDefaultPolicy
 	}
+	if b.EnableACLReplication {
+		result.EnableACLReplication = true
+	}
 	if b.ACLReplicationToken != "" {
```

Comment:
if they do a config reload with aclreplicationtoken set but `EnableACLReplication`set to false, does this still end up with `EnableACLReplication` set to true?